### DISCUSSION TITLE
To achieve Waterfall Collection view in xamarin forms

### DISCUSSION THREAD

**Discussion by praveennachi60** [xamarin/Xamarin.Forms#12295] [Q&A]
## Summary
To achieve Water fall collection view.

I can able in android using custom renderer. How to achieve this in ios renderer?

**Comment by hartez** [comment]
@praveennachi60  Can you describe what you mean by "waterfall"? Or provide some screenshots/mockups?

**Comment by praveennachi60** [comment]
> @praveennachi60 Can you describe what you mean by "waterfall"? Or provide some screenshots/mockups?

Sure @hartez . Ill share the screenshot.

**Comment by praveennachi60** [comment]
![waterfallcollection view](https://user-images.githubusercontent.com/61015267/94926042-cbe50700-04dd-11eb-8548-f8d209319dce.png)

**Comment by praveennachi60** [comment]
Like this above screenshot @hartez . Item will arrange in available spaces. I'd achieved this in android using staggered layout which they already provide. Unfortunately in iOS, I couldn't find a layout like this. 

I refer this github link to achieve the same.
https://github.com/xamarin/ios-samples/tree/master/ios9/CollectionView

But I don't know how to implement using custom renderer.

**Reply by shawyunz** [reply]
Similar to https://github.com/xamarin/Xamarin.Forms/issues/6554? But seems no solution yet.
